### PR TITLE
TERMINAL working

### PR DESCRIPTION
**Pull request by luroche**

### CODE REVIEW

File: code/TERMINAL/TERMINAL.py
Comment (mariuscrsn) [inline code/TERMINAL/TERMINAL.py:1]: Change name to TERM

File: code/TERMINAL/TERMINAL.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python3
+
+#######################        MANDATORY IMPORTS         #######################
+import sys, os
+sys.path.append(os.getcwd())  #get absolute path
+
+#######################      LOGGING CONFIGURATION       #######################
+from SYS.SYS_LOG import SYS_LOG_Logger_c, SYS_LOG_LoggerGetModuleLogger
+if __name__ == '__main__':
+    cycler_logger = SYS_LOG_Logger_c('./SYS/SYS_LOG/logginConfig.conf')
```
Comment (mariuscrsn) [inline code/TERMINAL/TERMINAL.py]: change `cycler_logger ` name to `root_logger`

File: code/TERMINAL/TERMINAL.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+#######################        MANDATORY IMPORTS         #######################
+import sys, os
+sys.path.append(os.getcwd())  #get absolute path
+
+#######################      LOGGING CONFIGURATION       #######################
+from SYS.SYS_LOG import SYS_LOG_Logger_c, SYS_LOG_LoggerGetModuleLogger
+if __name__ == '__main__':
+    cycler_logger = SYS_LOG_Logger_c('./SYS/SYS_LOG/logginConfig.conf')
+log = SYS_LOG_LoggerGetModuleLogger(__name__, config_by_module_filename="./log_config.yaml")
```
Comment (mariuscrsn) [inline code/TERMINAL/TERMINAL.py]: change path of log config gile to CONFIG folder. Add it (artifact) to ProjectStructure on IDEAS

File: code/TERMINAL/TERMINAL.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+#######################         GENERIC IMPORTS          #######################
+import time
```
Comment (mariuscrsn) [inline code/TERMINAL/TERMINAL.py]: Not use generic import.  Example: `from time import time`

File: code/TERMINAL/TERMINAL.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        for i in range(len(lines)):
+            Screen.clear()
+            print(padding + "\n".join(lines[:i]))
+            time.sleep(0.1)
+        time.sleep(3)
+        Screen.clear()
+        print(padding + intro_text)
+    
+    
+    def showProgressBar(self,iteration, total) -> None:
```
Comment (mariuscrsn) [inline code/TERMINAL/TERMINAL.py]: Add it to IDEAS

File: code/TERMINAL/__init__.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from .terminal import clear_terminal, print_intro, printProgressBar
+
+all = [
```
Comment (mariuscrsn) [inline code/TERMINAL/__init__.py]: Change all to public functions or classes: `all = [TERM_c, TERM_Option_e]`

File: code/TERMINAL/__init__.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from .terminal import clear_terminal, print_intro, printProgressBar
```
Comment (mariuscrsn) [inline code/TERMINAL/__init__.py]: Remove it and change to: `from .TERM import TERM_c, TERM_Option_e`

File: code/TERMINAL/TERMINAL.py (hunk 2 above)
Comment (luroche) [reply code/TERMINAL/TERMINAL.py]: done

**Comment by luroche** [comment]
All done

File: code/MANAGER/MANAGER.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+# #######################          PROJECT IMPORTS         #######################
+
+
+# #######################              ENUMS               #######################
+
+# #######################              CLASSES             #######################
+
+class MANAGER_c():
+    def __init__(self) -> None:
+        self._epc_config: STM_FLASH_Epc_Conf_c = None
```
Comment (mariuscrsn) [inline code/MANAGER/MANAGER.py:30]: change to __ if is private

File: code/MANAGER/MANAGER.py [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+        self._status: TERM_Option_e = None
+
+    def _calibStatus(self):
```
Comment (mariuscrsn) [inline code/MANAGER/MANAGER.py:33]: change to __

**Review by mariuscrsn — APPROVED** [review]

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**